doc/licenses/mybatis-guice-4.0.0/README

```diff
@@ -1,7 +1,7 @@
 MyBatis-Guice (http://www.mybatis.org/guice/)
 ---------------------------------------------
 
-    Version: 3.18
+    Version: 4.0.0
```

Comment:
This is a pretty big jump. Checking the dependencies for MyBatis-Guice 4.0.0, it looks like they may have migrated things from `javax.*` to `jakarta.*`, which might not be compatible if any of those classes will be provided by Tomcat. Can you confirm whether this is the case?

I haven't yet checked whether there are any dependencies that would break compatibility with Java 8, but that would be another thing to verify here.

Reply:
Ah yeah, good call. I'll revert this one.